### ISSUE TITLE
Package some magics to replace ophyd commands API

### ISSUE THREAD

**Issue by danielballan** [NSLS-II/bluesky#711]
Define magics for:

```
%mov motor1 5 motor2 6
%movr ...
%wh_pos
%wa  # synonym for wh_pos
%log_pos
```

**Comment by danielballan** [comment]
Those magics that depend on bluesky (i.e., use a RunEngine) are implemented in #720. Others only depend on ophyd and are addressed in https://github.com/NSLS-II/ophyd/pull/417

**Comment by danielballan** [comment]
In the end, none of the magics actually required an ophyd dependency, so they all went into #720. Closed by #720.
